### ISSUE TITLE
artifacts?

### ISSUE THREAD

**Issue by jloomb** [berkshelf/vagrant-berkshelf#297]
Am I meant to get a new unique 'berkshelf' in .berkshelf/vagrant-berkshelf/shelves/ ever time I vagrant up that remains after I vagrant destroy. Is there any options to prevent this as creating duplicate after duplicate whilst I'm developing means I need to manually watch/maintain the ever growing shelves folder?

**Comment by reset** [comment]
@jloomb yep, each virtual machine you begin to manage needs to have it's own self contained shelf. The contents of the shelf should be rather small and it should be cleaned up on destruction. If the shelves aren't being cleaned up then there may be some places where we aren't correctly cleaning up after ourselves
